config/config.go

```diff
@@ -392,10 +399,20 @@ func (sc *SamplerConfig) NewSampler(
 		options := []jaeger.SamplerOption{
 			jaeger.SamplerOptions.Metrics(metrics),
 			jaeger.SamplerOptions.InitialSampler(initSampler),
-			jaeger.SamplerOptions.SamplingServerURL(sc.SamplingServerURL),
 			jaeger.SamplerOptions.MaxOperations(sc.MaxOperations),
 			jaeger.SamplerOptions.OperationNameLateBinding(sc.OperationNameLateBinding),
-			jaeger.SamplerOptions.SamplingRefreshInterval(sc.SamplingRefreshInterval),
+		}
+		if sc.SamplerStrategiesFile != "" {
+			options = append(
+				options,
+				jaeger.SamplerOptions.SamplingStrategiesFile(sc.SamplerStrategiesFile),
+			)
+		} else {
+			options = append(
+				options,
+				jaeger.SamplerOptions.SamplingServerURL(sc.SamplingServerURL),
+				jaeger.SamplerOptions.SamplingRefreshInterval(sc.SamplingRefreshInterval),
```

Comment:
We should keep the refresh interval on the file option.  It will allow people to dynamically swap out the sampling file w/o restarting their service.

Reply:
Oh I originally intended that, didn't catch this, thanks!